frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/OverrideForkPointDialog.kt

```diff
@@ -87,6 +91,42 @@ class OverrideForkPointDialog(
             thisRowComment
           )
       }
+
+      row {
+        radioButton(
+          format(
+            getString(
+              "action.GitMachete.BaseOverrideForkPointAction.dialog.override-fork-point.radio-button.custom"
+            ),
+            radioButtonComment
+          ),
+          OverrideOption.CUSTOM
+        )
+
+        val mutableList = branch.commitsUntilParent.toMutableList()
+        mutableList.remove(branch.forkPoint)
+        mutableList.remove(parent.pointedCommit)
+        val items = (listOf(branch.forkPoint, parent.pointedCommit) + mutableList.reversed()).filterNotNull()
+
+        comboBox(
+          items,
+          object : DefaultListCellRenderer() {
+            override fun getListCellRendererComponent(
+              list: JList<*>?,
+              value: Any?,
+              index: Int,
+              isSelected: Boolean,
+              cellHasFocus: Boolean
+            ): Component {
+              val commit: ICommitOfManagedBranch = value as ICommitOfManagedBranch
+              text = "<html><tt>[${commit.shortHash}]</tt> ${commit.shortMessage}</html>"
```

Comment:
<img width="961" alt="image" src="https://user-images.githubusercontent.com/19799111/196155334-7325c15f-2827-44e3-b51b-c18fc1376745.png">

monospace font for commit hashes - same len of them, better look imho

Reply:
Lemme comment here so that it's easier to track:

Wow _it's so f**king ~big~ wide_:

![image](https://user-images.githubusercontent.com/3383210/196203305-e238b64f-37f2-4d59-a5be-207dcc9199e5.png)

Use commit subject, not full message

Reply:
Although huh, it's `shortMessage`... hm WTF

Reply:
Awww I see, check `git show 2a09`:

![image](https://user-images.githubusercontent.com/3383210/196205149-00b6de5c-26c6-406d-8ddd-095317cdb49f.png)

and now check the javadoc for `org.eclipse.jgit.revwalk.RevCommit#getShortMessage`:

![image](https://user-images.githubusercontent.com/3383210/196205247-4e7cf6f6-2e88-41dd-b6f2-9e3306d3cce7.png)

It's a general convention that the **second line** of commit message, if exists, is always empty (as a separating between subject and extended description).

However, judging by the following line in the implementation of `getShortMessage`:

```java
                if (hasLF(raw, msgB, msgE)) {
			str = StringUtils.replaceLineBreaksWithSpace(str);
		}
```

they're indeed aware that the subject (part until first double LF) itself might be multiline.

Reply:
#1277